sql/core/src/test/scala/org/apache/spark/sql/streaming/StreamSuite.scala

```diff
@@ -381,7 +390,7 @@ class StreamSuite extends StreamTest {
 
   test("insert an extraStrategy") {
     try {
-      spark.experimental.extraStrategies = TestStrategy :: Nil
+      spark.experimental.extraStrategies = CustomStrategy :: Nil
```

Comment:
Since we need to do a temporary planning for streaming queries, we can't allow custom strategy to remove streaming leaf nodes.